src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -93,20 +94,29 @@ export function PreviewSiteRow( {
 			<div className="flex items-center px-8 py-6">
 				<div className="w-[51%]">
 					<div className="flex items-center">
-						<div className="text-[13px] leading-5 line-clamp-1 break-all">
+						<div
+							className={ cx(
+								'text-[13px] leading-5 line-clamp-1 break-all',
+								isExpired && 'line-through text-[#757575]'
```

Comment:
I've investigated if we have the color `text-a8c-gray-700` - `#757575` , but it seems we don't have it. I've tried updating `automattic/color-studio` but the new version 4.1.0 doesn't include it either.
I think it's ok to leave it as it is, since we are using that color in the rest of the app, but we could consider adding a new variable in [tailwind config](https://github.com/Automattic/studio/blob/710737ebd8278291ff87b26fd7539cfc8a704779/tailwind.config.js#L134).

Something like:

`a8cToTailwindColors[ `${ PREFIX }-gray-700` ] = '#757575'; // Gray 700`

Reply:
That's a great idea! Thanks! I have added the variable and replaced all occurrences of `#757575` in the latest commit: https://github.com/Automattic/studio/pull/913/commits/323002b5db0adc581468f03d5bb5f6682ec2f69e.